ase/scripts/generate_ase_environment.py

```diff
@@ -478,8 +505,9 @@ def gen_afu_json_verilog_macros(json_file):
                             will be parsed by rtl_src_config.""")
 parser.add_argument('-t', '--tool', choices=['VCS', 'QUESTA'], default=None,
                     help='simulator tool to use, default is VCS if present')
-parser.add_argument('-p', '--platform', choices=['intg_xeon', 'discrete'],
-                    default='intg_xeon', help='FPGA Platform to simulate')
+parser.add_argument('-p', '--platform',
+                    default=get_default_platform(),
+                    help='FPGA Platform to simulate')
 parser.add_argument('-x', '--exclude', default=None,
```

Comment:
can we add an option to list all supported platforms?

Reply:
afu_platform_config --help